### ISSUE TITLE
Expectation autocomplete no longer showing up in tests

### ISSUE THREAD

**Issue by roni-estein** [pestphp/pest-intellij#492]
**Describe the bug**
I've taken some time away from code, so things may have changed. I got through updating my system apple M1 with brand new phpstorm, and all updated plugins.

I've created a laravel 10 project and installed pest per site guidelines eager to try out all the new features of pest 2.0. I have noticed  that things that I believe used to simply work, specifically code autocompletion for expectations no longer appear to function. I did notice someone had a windows issue that looked similar, but it did not resolve anything for my setup. I don't have the "idea" plugin that seems to be the rage at the moment, and it also looks like barryvdh/idehelper is not working for laravel 10 per issues on that repo, only mentioning because they may be contributing factors.

**To Reproduce**
1. Install a new laravel project
2. install pest with laravel plugins (I added all to be sure)
3. create a test
4. write an expectation and expect autocomplete to work.

**Expected behavior**
pest expectations should appear while typing

```php
//for example when typing this out
expect(1)->toBeOne();

//at this point options used to appear
expect(1)-> //a context dropdown should list the associated methods.

//however nothing happens. As a test I left it on all night.

```

Here some screen shots
**Screenshots**
This works as expected, not the same issue referenced from the [issue #457](https://github.com/pestphp/pest-intellij/issues/457)
![Screenshot 2023-08-23 at 3 41 19 PM](https://github.com/pestphp/pest-intellij/assets/8517475/8a40e4ba-c3c0-4d89-8b54-1727abae0434)

Also as expected
![Screenshot 2023-08-23 at 3 42 30 PM](https://github.com/pestphp/pest-intellij/assets/8517475/efc792fa-368e-433e-be04-22ae1c2c7d35)

This is where it diverges from what I remember and sadly the most useful part.
![Screenshot 2023-08-23 at 3 42 52 PM](https://github.com/pestphp/pest-intellij/assets/8517475/1f13ca77-e025-4074-8b6d-a1db499d98a0)

**Desktop (please complete the following information):**
 - MacOS Sonoma 14.0
 - IntelliJ Version: PhpStorm 2023.2.1 RC Build #PS-232.9559.43, built on August 18, 2023
 - Plugin Version 1.10.0-EAP.1
 - php 8.2.9
 - laravel Herd 1.2.0 (build 14)

**Additional context**
This hasn't appeared to be working for a some time for me, hence the updating of everything to the newest possible versions

**Comment by olivernybroe** [comment]
Hey 👋 
Thanks for reporting, this is a duplicate of #489 which turned out to be a bug in phpstorm with generics :) 
https://youtrack.jetbrains.com/issue/WI-73663

**Comment by roni-estein** [comment]
Thanks! Sorry I missed it before and made a new issue.Sent from my iPhoneOn Aug 24, 2023, at 5:37 AM, Oliver Nybroe ***@***.***> wrote:﻿
Closed #492 as completed.
